src/components/EditCollectiveForm.js

```diff
@@ -43,6 +44,8 @@ const MenuItem = styled(Link)`
   ${({ selected }) => selected && selectedStyle};
 `;
 
+const archiveIsEnabled = getEnvVar('SHOW_ARCHIVE_COLLECTIVE');
```

Comment:
Needs to be `parseToBoolean`, there is function in `lib/utils.js` for that.

Reply:
Oh.. Alright